packages/ui/src/containers/AuthContainer.ts

```diff
@@ -159,8 +159,22 @@ export class AuthContainer {
     let form = this.accountForm!;
     if (form instanceof NewAccountFormData && form.clean()) {
       // Save the private and public keys to disk.
-      saveToFile(form.privateKeyBase64, `${form.name.$}.private.key`);
-      saveToFile(form.publicKeyBase64.$!, `${form.name.$}.public.key`);
+      saveToFile(
+        Keys.Ed25519.privateKeyEncodeInPem(decodeBase64(form.privateKeyBase64)),
+        `${form.name.$}-account-private.pem`
+      );
+      saveToFile(
+        Keys.Ed25519.publicKeyEncodeInPem(
+          decodeBase64(form.publicKeyBase64.$!)
+        ),
+        `${form.name.$}-account-public.pem`
+      );
+      saveToFile(
+        encodeBase16(
+          Keys.Ed25519.publicKeyHash(decodeBase64(form.publicKeyBase64.$!))
```

Comment:
This doesn't match up with what the Rust client generates.  We are also encoding the key type as the first byte, so for Ed25519 as hex, we're prefixing the key's bytes with `01`.

Reply:
I would check it one hour later, creating another PR now.